python/requirements/ml/data-requirements.txt

```diff
@@ -6,6 +6,7 @@ aioboto3==11.2.0
 crc32c==2.3
 flask_cors
 modin==0.22.2
-raydp>=0.0.dev0
+pandas==1.5.3
```

Comment:
is the pandas pin required for data specifically?

Reply:
i was just helping out pip a little bit in its quest to resolve dependencies, not specific to data; 1.5.3 is essentially the latest 1.* version of pandas